src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -53,21 +55,41 @@ TaskInfo create(String uid, String primaryKey) throws MeilisearchException {
      * Gets an index from its uid
      *
      * @param uid Unique identifier of the index to get
-     * @return Meilisearch API response
+     * @return Meilisearch API response as Index instance
      * @throws MeilisearchException if an error occurs
      */
-    String get(String uid) throws MeilisearchException {
-        String requestQuery = "/indexes/" + uid;
-        return httpClient.get(requestQuery, String.class);
+    Index getIndex(String uid) throws MeilisearchException {
+        return httpClient.get(new IndexesQuery().toQuery(uid), Index.class);
     }
 
     /**
      * Gets all indexes in the current Meilisearch instance
```

Comment:
```suggestion
     * Gets indexes in the current Meilisearch instance
```

I removed the `all` because they are now returned paginated thus they might not all be returned